apps/showcase/src/components/showcase/basic/basic-pres.template.html

```diff
@@ -1,7 +1,7 @@
 <div class="card my-3">
   <div class="row g-0">
     <div class="col-4 bg-body-tertiary d-flex align-items-center justify-center">
-      <img src="assets/otter.svg" class="img-fluid rounded-start" aria-hidden="true">
+      <img alt="Otter image" src="assets/otter.svg" class="img-fluid rounded-start" aria-hidden="true">
```

Comment:
Maybe we could add some context on why we have this otter here?
```suggestion
      <img alt="Otter image that may be replaced depending on user inputs" src="assets/otter.svg" class="img-fluid rounded-start" aria-hidden="true">
```